js/map_utils.js

```diff
@@ -4,6 +4,11 @@
 var nationDims;
 var zoom_scale;
 
+function extractNames(stateBounds) {
+  for (var i = 0, len = stateBounds.features.length; i < len; i++) {
+    waterUseViz.stateAbrvs.push(stateBounds.features[i].properties.STATE_ABBV);
+  }
+}
```

Comment:
there is a `forEach` that allows you to clean this up. I haven't had too much experience with it, but I played with it and you can do this:

```
stateBounds.features.forEach(function(d) {
     waterUseViz.stateAbrvs.push(d.properties.STATE_ABBV);
});
```